src/client/components/input/drag-and-drop-image.js

```diff
@@ -14,6 +29,11 @@ class DragAndDropImage extends React.Component {
 		};
 		ev.dataTransfer.setData('text', JSON.stringify(data));
 	}
+	/**
+	* Renders an image of the achievement badges which supports
+	* DragAndDrop functionality.
+	* @returns {ReactElement} a HTML image supporting DragAndDrop functionality.
```

Comment:
`the rendered image element` works better here (more consistent with other React `@returns` comments, and avoids repeating the part about DragAndDrop functionality).